first session/assignment/shoppingmall.css

```
@@ -0,0 +1,126 @@
+body {
+    font-family: Arial, sans-serif;
+}
+
+.hamburger-menu .menu {
+    display: none;
+}
+
+
+.hamburger-icon {
+    background: none;
+    border: none;
+    font-size: 24px;
+    cursor: pointer;
+    padding: 10px;
+}
+
+header{
+    display: flex;
+    justify-content: space-between;
+    padding: 10px;
+    position: fixed;
+    width: 100%;
+    top: 0;
+}
+header .menu-bar {
+    width: 20px;
+    font-size: 500px;
+}
+
+main {
+    width: 80%;
+    margin: 0 auto; 
+}
+
+/* 운동용품 섹션의 상품들을 Flexbox를 사용하여 가로로 정렬 */
+#category1 {
+    display: flex;
+    flex-wrap: wrap; /* 상품들이 너비를 초과할 경우 다음 줄로 넘어가도록 설정 */
+    gap: 20px; /* 상품 사이의 간격 설정 */
+}
+#category2 {
+    display: flex;
+    flex-wrap: wrap; /* 상품들이 너비를 초과할 경우 다음 줄로 넘어가도록 설정 */
+    gap: 20px; /* 상품 사이의 간격 설정 */
+}
+
+#category3 {
+    display: flex;
+    flex-wrap: wrap; /* 상품들이 너비를 초과할 경우 다음 줄로 넘어가도록 설정 */
+    gap: 20px; /* 상품 사이의 간격 설정 */
+}
+#all {
+    display: flex;
+    flex-wrap: wrap; /* 상품들이 너비를 초과할 경우 다음 줄로 넘어가도록 설정 */
+    gap: 20px; /* 상품 사이의 간격 설정 */
+}
```

요 부분들은 지금 동일한 속성들을 공유하고 있죠?? id를 사용하면 중복선언이 안 되기 때문에 동일 속성을 여러 태그에 적용시킬 수 없다는 단점이 있어요!!

아마 그런데 id를 사용한 이유가 a태그 이동을 위해서였을거예요!!

하나의 태그에는 id와 class를 동시에 지정할 수 있어요!! id는 그대로 두고 동일한 스타일을 적용할 태그의 class를 모두 통일시킨 다음, css에는 해당하는 class에 대한 스타일만 1번 작성하면 됩니다!!

```suggestion
.category {
    display: flex;
    flex-wrap: wrap; /* 상품들이 너비를 초과할 경우 다음 줄로 넘어가도록 설정 */
    gap: 20px; /* 상품 사이의 간격 설정 */
}
```

이렇게 하고 위의 저 4개 id값이 지정되어있는 태그에서 class로 "category"를 추가하면 되겠죠!!